core/src/apps/bitcoin/sign_tx/approvers.py

```diff
@@ -113,6 +116,7 @@ def add_orig_change_output(self, txo: TxOutput) -> None:
 
     async def add_external_output(
         self,
+        tx_info: TxInfo,
```

Comment:
If I understand this correctly, the reason why we need to add the TxInfo here is so that we can show the source account if the user presses the info button. But do we really want that screen to be accessible from the output confirmation dialog? Shouldn't it just be accessible from the Summary dialog?

Reply:
Why it shouldn't be accessible from every screen?

Reply:
It introduces a new dependency into the output confirmation and adds some complexity, which is always a reason to stop and think whether it's justified, because it can lead to more complexity in the long run. But I don't see any significant problem with it.

Reply:
Also the button takes up room which could be better used to clearly communicate the account name when the output is to another of the user's own accounts.

Reply:
Ok, we can put it only on the last screen if you wish.

There is unfortunately not enough space for it: 
<img width="353" alt="image" src="https://user-images.githubusercontent.com/57008159/229303898-64017f78-ab5a-4726-a119-6908f247d4eb.png">

Reply:
Removed transaction details from confirm output layout, as well as passing `tx_info` around, in ba5f6490a6e802aa97a2441baef26decb8560c47.

Reply:
BTW, I just realized one nice example of how "it can lead to more complexity in the long run". If you want to add the fee rate to the info screen, then you wouldn't be able to do that in the output confirmation dialogue, because the fee is not known yet. It would require adding a new phase to the signing flow.